web/index.php

```diff
@@ -153,6 +150,29 @@
     return $app['twig']->render('search.twig', $data);
 });
 
+$app->post('/update', function (Request $request) use ($app, $searchForm) {
+    // first run the update command
+    $name = $request->get('name');
+    if (!trim($name)) {
+        return new Response('Invalid Request',400);
+    }
+
+    $count = getRequestCountByIp($_SERVER['REMOTE_ADDR'], $app['db']);
+    if ($count > 10) {
+        return new Response('Too many requests. Try again in an hour.', 403);
+    }
+
+    $input = new ArrayInput(array(
+        'command' => 'update',
+        '--name' => $name
```

Comment:
Maybe match against the slug in the DB first using a prepared statement? And then if slug exists run the command on the returned slug rather than the user input. I'm not sure it's safe to just pass the name here.

Reply:
Good idea, I've done that now